cloud/scope/powervs_cluster_test.go

```diff
@@ -3581,7 +3731,7 @@ func TestSetVPCSubnetStatus(t *testing.T) {
 					Status: infrav1beta2.IBMPowerVSClusterStatus{
 						VPCSubnet: map[string]infrav1beta2.ResourceReference{
 							"subnet1Name": {
-								ControllerCreated: ptr.To(false),
+								ControllerCreated: ptr.To(true),
```

Comment:
nit: why we need this change?

Reply:
Not required, have reverted

Reply:
I had to cover the case where the controllercreated value is true- https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/48777461bd36a5ca27d61ffc6e83af5163ae95f6/cloud/scope/powervs_cluster.go#L529